src/test/java/org/sagebionetworks/bridge/sdk/integration/SubpopulationTest.java

```diff
@@ -77,6 +79,7 @@ public void canCRUD() {
         } catch(BadRequestException e) {
             assertEquals("Cannot delete the default subpopulation for a study.", e.getMessage());
         }
+        admin.getSession().getAdminClient().deleteSubpopulationPermanently(retrieved.getGuid());
```

Comment:
We should move this to a try-finally or to an After method. Otherwise, if the Subpop test fails, we won't clean up subpops. This will cause cascading failures down the line.